actor/v7pushaction/sequence.go

```diff
@@ -21,7 +21,7 @@ func ShouldStagePackage(plan PushPlan) bool {
 }
 
 func ShouldCreateDeployment(plan PushPlan) bool {
-	return plan.Strategy == constant.DeploymentStrategyRolling
+	return plan.Strategy == constant.DeploymentStrategyRolling || plan.Strategy == constant.DeploymentStrategyCanary
```

Comment:
*nit* Do we need to enumerate all strategies here? Is there a reason to not just check strategy is != ""?

Reply:
Done